journey-map/public/index.html

```diff
@@ -14,6 +14,7 @@ <h1>
                     </svg>
                     <span>IPV Core Journey Map</span>
                 </h1>
+                <input id="state-search" type="text" placeholder="Search for state...">
```

Comment:
Could consider making this a `search` type (if you haven't already), get a few minor UX things for free:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/search#differences_between_search_and_text_types

Reply:
Sounds good, will update it